taskt/UI/Forms/frmScriptBuilder.Designer.cs

```diff
@@ -90,7 +90,7 @@ private void InitializeComponent()
             this.grpSaveClose = new taskt.UI.CustomControls.UIGroupBox();
             this.btnSequenceImport = new taskt.UI.CustomControls.UIPictureButton();
             this.uiBtnKeep = new taskt.UI.CustomControls.UIPictureButton();
```

Comment:
Similarly, remove `taskt.UI` and other namespace references.

Reply:
So we are not going to update auto-generated code of Designer Classes (as decided).